sdk/communication/communication-jobrouter/samples-dev/DistributionPolicyCrud.ts

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+/**
+ * @summary Distribution policy crud
+ */
+// Load the .env file (you will need to set these environment variables)
+import * as dotenv from "dotenv";
+import { DistributionPolicy } from "../src";
+import { DistributionPolicyItem } from "../src";
```

Comment:
You should import from `@azure/communication-jobrouter` instead of `../src`. Also you can import all of them in the same line

Reply:
Since the package is not available on npm azure registry yet, it will cause build errors. I think I can update these import before release, together with the date update in ChangeLog.

Reply:
Our samples infrastructure handles package linking and will be able to resolve this as a local dependency. Think of samples as code that will be copy/pasted by customers to get familiar with the service and the library. These references are not portable